### ISSUE TITLE
Bug in Register Shared Bundle - Deploy Main Github CI/CD

### ISSUE THREAD

**Issue by slawpast** [microsoft/AzureTRE-Deployment#58]

**Comment by marrobi** [comment]
This would have avoided the issue.... https://github.com/microsoft/AzureTRE/issues/2727

**Comment by tamirkamara** [comment]
> This would have avoided the issue.... [microsoft/AzureTRE#2727](https://github.com/microsoft/AzureTRE/issues/2727)

I don't think this is actually related. We are testing each version before we create the release. The issue here is that we don't have workflows (CICD) running in the deployment repo.
